### PR TITLE
Output: Call get() function with parameter names

### PR DESCRIPTION
**Pull request by jan-janssen**

### CODE REVIEW

File: atomistics/shared/output.py [hunk 1]
```diff
@@ -7,8 +7,8 @@ class Output:
     def keys(cls):
         return tuple(field.name for field in dataclasses.fields(cls))
 
-    def get(self, engine, *output: str) -> dict:
-        return {q: getattr(self, q)(engine) for q in output}
+    def get(self, engine, output_keys: str) -> dict:
+        return {q: getattr(self, q)(engine) for q in output_keys}
```
Comment (liamhuber) [inline atomistics/shared/output.py:11]: I like changing the name from `output` to `output_keys`, but I dislike changing it from `*args` to an iterable. Additionally, this change leaves the type hint invalid.

**Review by liamhuber — COMMENTED** [review]
I see post change that you can (and do) specify the entire argument set with kwargs, even when not necessary (which is a perfectly fine stylistic choice). Is there some forward-looking benefit to this capability? If not I actually dislike changing the signature of `get` from `x, *y` to `x, y`.

**Comment by jan-janssen** [comment]
> I see post change that you can (and do) specify the entire argument set with kwargs, even when not necessary (which is a perfectly fine stylistic choice). Is there some forward-looking benefit to this capability? If not I actually dislike changing the signature of `get` from `x, *y` to `x, y`.

The way I see it, we have functions in the individual codes, like `calc_md()` or `calc_static()`, which implement the interface as:
```
def calc_static(..., output_keys):
    return OutputStatic(
        **{k: getattr(Interface, k) for k in OutputStatic.keys()}
    ).get(
       Interface(),
        *output_keys,
    )
```
Then inside the `OutputStatic` interface we loop over the variable:  
```
def get(self, engine, *output: str) -> dict:
        return {q: getattr(self, q)(engine) for q in output}
```

As the user facing part is using a regular variable without `*` like `output_keys` I see no reason why the internal interface should use the `*output_keys` if internally it is anyway just used to iterate over the variable. 

From my perspective it is essential for us to define consistent interfaces and not just once use once `*output_keys` and the other time `output_keys`. That is just confusing.

**Review by liamhuber — APPROVED** [review]
> As the user facing part is using a regular variable without `*` like `output_keys` I see no reason why the internal interface should use the `*output_keys` if internally it is anyway just used to iterate over the variable.
>
> From my perspective it is essential for us to define consistent interfaces and not just once use once `*output_keys` and the other time `output_keys`. That is just confusing.

Ok, so it is just a style/UX thing.

I disagree and don't find it confusing, as the signatures are for different classes. But it's just a preference I hold weakly and I don't have any objection to the new form, so go for it.

**Pull request merged**